### PR TITLE
Update School Data Sync OneRoster connection settings

### PR DESCRIPTION
**Pull request by abhijitsverma**
Update School Data Sync OneRoster connection settings

### CODE REVIEW

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# educationSynchronizationConnectionSettings resource type
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Global: Please change the filenames to use all lowercase, not camel casing. Mixed case filenames result in broken links on our site.

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Represents the provider connection settings. This allows the system to know how to connect to the provider APIs. 
+
+> **Note:** This complex type is abstract. Refer to the specific types of connection settings listed.
+
+## Derived types
+| Type | Description | 
+|:-|:-|
+| [**educationSynchronizationOAuth1ConnectionSettings**](educationSynchronizationOAuth1ConnectionSettings.md) | Use this type to provide OAuth1 connection settings |
+| [**educationSynchronizationOAuth2ClientCredentialsConnectionSettings**](educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md) | Use this type to provide OAuth2 Client Credentials Grant connection settings |
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Global: Add periods to the ends of the sentences in the Description column.

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Properties
+
+| Property | Type | Description |
+|:-|:-|:-|
+| **clientId** | String |  Client id used to connect to the provider |
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Change id to ID.

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+| **clientSecret** | String |  Client secret to authenticate connection to the provider |
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Change to "authenticate the connection".

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# educationSynchronizationOAuth1ConnectionSettings resource
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+When OAuth1 is to be used to connect to the data provider, this connection settings type should be used to setup the profile.
+
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Change "setup" to "set up" (two words).

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# educationSynchronizationOAuth1ConnectionSettings resource
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+When OAuth1 is to be used to connect to the data provider, this connection settings type should be used to setup the profile.
+
+Derived from [educationSynchronizationConnectionSettings](educationSynchronizationConnectionSettings.md)
+
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Add a period.

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+When OAuth1 is to be used to connect to the data provider, this connection settings type should be used to setup the profile.
+
+Derived from [educationSynchronizationConnectionSettings](educationSynchronizationConnectionSettings.md)
+
+## Properties
+
+No additional properties are exposed by this type
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Add a period.

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+## JSON
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Change the H2 heading to "JSON representation". Also, add the HTML comment block to validate the JSON.

File: api-reference/beta/resources/educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+# educationSynchronizationOAuth2ClientCredentialsConnectionSettings resource
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+When [OAuth2 Client Credentials Grant](https://tools.ietf.org/html/rfc6749#section-4.4) is to be used to connect to the data provider, this connection settings type should be used to setup the profile.
+
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md]: "set up".

File: api-reference/beta/resources/educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+Derived from [educationSynchronizationConnectionSettings](educationSynchronizationConnectionSettings.md)
+
+## Properties
+
+| Property | Type | Description |
+|:-|:-|:-|
+| **tokenUrl** | String | The URL to get access tokens for the data provider |
+| **scope** | String | [The scope of the access request](https://tools.ietf.org/html/rfc6749#section-3.3) |
+
+## JSON
```
Comment (Lauragra) [inline api-reference/beta/resources/educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md]: This should be "JSON representation", and needs to include the HTML comment block.

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md (hunk 2 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md (hunk 3 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationConnectionSettings.md (hunk 4 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md (hunk 5 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md (hunk 6 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md (hunk 7 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md (hunk 8 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationOAuth1ConnectionSettings.md]: Done

File: api-reference/beta/resources/educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md (hunk 9 above)
Comment (abhijitsverma) [reply api-reference/beta/resources/educationSynchronizationOAuth2ClientCredentialsConnectionSettings.md]: Done

**Pull request merged**